centreon/src/Core/Command/Application/UseCase/GetCommand/GetCommand.php

```
@@ -0,0 +1,196 @@
+use Core\Application\Common\UseCase\NotFoundResponse;
+use Core\Security\AccessGroup\Domain\Model\AccessGroup;
+use Core\Command\Domain\Model\Argument;
+use Core\Command\Domain\Model\CommandType;
+use Core\CommandMacro\Domain\Model\CommandMacro;
+
+final class GetCommand
+{
+    use LoggerTrait;
+
+     /** @var AccessGroup[] */
+    private array $accessGroups;
```

PHPMD (2.15.0)

Avoid unused private fields such as '$accessGroups'. (undefined)

(UnusedPrivateField)